content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,253 @@
+# dvc.api.params_show()
+
+Returns all the <abbr>params</abbr> associated with the given <abbr>DVC
+repository</abbr>.
+
+```py
+def params_show(
+    *targets: str,
+    repo: Optional[str] = None,
+    stages: Optional[Union[str, Iterable[str]]] = None,
+    rev: Optional[str] = None,
+    deps: bool = False,
+) -> Dict:
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.params_show()
+```
+
+## Description
+
+Retrieves the <abbr>params</abbr> tracked in a <abbr>DVC repository</abbr>.
+
+Without arguments, this function will retrieve all <abbr>params</abbr> from all
+parameter files, for the current revision.
```

Comment:
What about unstaged changes? Does this read the actual files in the working tree, or the ones in `HEAD`?

Reply:
> Does this read the actual files in the working tree, or the ones in HEAD?

It reads the actual files in the working tree. Should we replace `current revision` with `working tree` or `workspace`?

Reply:
I replaced with `working tree`